### PR TITLE
Added transitions 'VerticalUpSwipeJump' and 'VerticalDownSwipeJump' in…

### PR DESCRIPTION
**Pull request by vorjdux**
… NavigatorSceneConfigs

### CODE REVIEW

**Comment by vorjdux** [comment]
Hi,

I made one video for show the animation

https://drive.google.com/file/d/0BwY3eLp5Z-sccjlaQUtZMlFBVGc/view?usp=sharing

Best regards!

**Comment by vorjdux** [comment]
Any news?

**Comment by vorjdux** [comment]
@vjeux something wrong in this code?

Best regards.

File: Libraries/CustomComponents/Navigator/NavigatorSceneConfigs.js [hunk 1]
```diff
@@ -221,6 +271,50 @@ var FromTheLeft = {
...
+    to: 0,
+    min: 0,
+    max: 1,
+    type: 'linear',
+    extrapolate: true,
+    round: PixelRatio.get(),
+  },
+};
+
+var FromTheUp = {
```
Comment (ericvicenti) [inline Libraries/CustomComponents/Navigator/NavigatorSceneConfigs.js]: Maybe `FromTheTop` instead?

Comment (vorjdux) [inline Libraries/CustomComponents/Navigator/NavigatorSceneConfigs.js]: Yes, it gets better! I will change, thank you.

**Comment by vorjdux** [comment]
@ericvicenti done. I changed the interpolator name.

**Comment by ericvicenti** [comment]
@facebook-github-bot #import

**Comment by vjeux** [comment]
@facebook-github-bot import

**Pull request closed without merging**